src/validators/satelliteIdValidator.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, response } from 'express'
+import { assert, pattern, string } from 'superstruct'
+
+export const satelliteIdRouteValidation = (req: Request, res: Response, next: CallableFunction): void | ReturnType<typeof response.sendStatus> => {
```

Comment:
Change `CallableFunction` to `NextFunction` (import from express), and if you like, `void` to `ReturnType<NextFunction>`.